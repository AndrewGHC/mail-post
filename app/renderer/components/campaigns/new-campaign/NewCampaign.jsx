// @flow
import React from 'react';
import { csvImport } from '../../../utils';

const NewCampaign = () => (
  <div>
    <button onClick={() => csvImport()}>
      Import CSV
    </button>
  </div>
);

export default NewCampaign;
