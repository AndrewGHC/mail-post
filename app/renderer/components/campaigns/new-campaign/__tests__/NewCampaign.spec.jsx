import React from 'react';
import { shallow } from 'enzyme';

import NewCampaign from '../NewCampaign';

describe('NewCampaign', () => {
  it('renders elements', async () => {
    const wrapper = shallow(<NewCampaign />);
    expect(wrapper).toMatchSnapshot();
  });
});
