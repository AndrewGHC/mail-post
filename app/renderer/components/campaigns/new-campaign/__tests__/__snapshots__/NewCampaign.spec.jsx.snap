// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewCampaign renders elements 1`] = `
<div>
  <button
    onClick={[Function]}
  >
    Import CSV
  </button>
</div>
`;
